### PR TITLE
move implementation details from admin guide closer to code

### PR DESCRIPTION
**Pull request by xbaran4**
Signed-off-by: xbaran4 <pbaran@redhat.com>

> Read our [Contribution guide](https://github.com/eclipse/che-docs/blob/master/CONTRIBUTING.adoc) before submitting a PR.

### What does this PR do?
Removes sections from documentation that are not very useful for admins. 
These section are to be moved closer to code. https://github.com/eclipse/che/pull/19285

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/17266

### Specify the version of the product this PR applies to.
Latest.

### PR Checklist

As the author of this Pull Request I made sure that:

- [x] `vale` has been run successfully against the PR branch
- [x] Link checker has been run successfully against the PR branch
- [ ] Documentation describes a scenario that is already covered by QE tests, otherwise an issue has been created and acknowledged by Che QE team
- [ ] Changed article references are updated where they are used (or a redirect has been set up on the docs side):
    - [ ] Dashboard [branding.json](https://github.com/eclipse/che-dashboard/blob/master/src/components/branding/branding.json)
    - [ ] Chectl [constants.ts](https://github.com/che-incubator/chectl/blob/master/src/constants.ts)

### CODE REVIEW

**Review by yhontyk — APPROVED** [review]

File: modules/administration-guide/partials/proc_authenticating-to-the-che-server-using-openid.adoc [hunk 1]
```diff
@@ -1,109 +0,0 @@
...
-[source,json]
-----
-{
-    "che.keycloak.token.endpoint": "http://172.19.20.9:5050/auth/realms/che/protocol/openid-connect/token",
-    "che.keycloak.profile.endpoint": "http://172.19.20.9:5050/auth/realms/che/account",
-    "che.keycloak.client_id": "che-public",
-    "che.keycloak.auth_server_url": "http://172.19.20.9:5050/auth",
-    "che.keycloak.password.endpoint": "http://172.19.20.9:5050/auth/realms/che/account/password",
-    "che.keycloak.logout.endpoint": "http://172.19.20.9:5050/auth/realms/che/protocol/openid-connect/logout",
-    "che.keycloak.realm": "che"
```
Comment (ibuziuk) [inline modules/administration-guide/partials/proc_authenticating-to-the-che-server-using-openid.adoc:35]: @xbaran4 those properties are no longer relevant?

File: modules/administration-guide/partials/proc_authenticating-to-the-che-server-using-openid.adoc [hunk 2]
```diff
@@ -1,109 +0,0 @@
...
--d "subject_token=__<USER_OPENSHIFT_TOKEN>__" \ <1>
--d "subject_issuer=__<OPENSHIFT_IDENTITY_PROVIDER_NAME>__" \ <2>
---data-urlencode "grant_type=urn:ietf:params:oauth:grant-type:token-exchange" \
---data-urlencode "subject_token_type=urn:ietf:params:oauth:token-type:access_token" \
-https://__<KEYCKLOAK_HOST>__/auth/realms/{prod-deployment}/protocol/openid-connect/token <3>
-----
-<1> The token retrieved by the end-user with the command `oc whoami --show-token`
-<2> `openshift-v4` for OpenShift 4.x and `openshift-v3` for OpenShift 3.11
-<3> {identity-provider} host
-
```
Comment (ibuziuk) [inline modules/administration-guide/partials/proc_authenticating-to-the-che-server-using-openid.adoc:107]: cc: @tolusha I thought this was one of the top common question  - how to retrieve the keycloak token based on OpenShift token.

@xbaran4 what is the reason behind the removal of this section?

**Comment by ibuziuk** [comment]
@xbaran4 from the PR description it is not at all clear that docs are moved as-is to https://github.com/eclipse/che/pull/19285/files

File: modules/administration-guide/partials/proc_authenticating-to-the-che-server-using-openid.adoc (hunk 2 above)
Comment (xbaran4) [reply modules/administration-guide/partials/proc_authenticating-to-the-che-server-using-openid.adoc:107]: https://github.com/eclipse/che/issues/17266
TLDR; @skabashnyuk thinks that these section are more helpful for Che developers rather than admins and they will be moved closer to the code itself. https://github.com/eclipse/che/pull/19285

**Comment by xbaran4** [comment]
> @xbaran4 from the PR description it is not at all clear that docs are moved as-is to https://github.com/eclipse/che/pull/19285/files
Yea, this PR was created firstly as a draft, will edit it in.

**Review by ibuziuk — APPROVED** [review]

**Review by MichalMaler — APPROVED** [review]
